packages/mobile/src/components/Avatar.tsx

```diff
@@ -0,0 +1,84 @@
+/**
+ * Essentially the same as @celo/react-components/components/Avatar but with
```

Comment:
Funny, it so happens I also wrapped the react-components avatar in my PR here: https://github.com/celo-org/celo-monorepo/pull/831
But for a slightly different reason. I wanted to connect in the `defaultCountryCode` prop so that consumers of this didn't have to do it.